docs/porting-addons-to-v2.md

```diff
@@ -125,6 +128,14 @@ The steps:
    ember s
    ember test
    ```
+
+1. The lint scripts are expected to work the same way as before inside the test-app. However, there's one common issue you may encounter when running the linter if you use `eslint-plugin-n` or `eslint-plugin-node`: 
```

Comment:
Do you have a repro of this? It kinda feels like we should look at fixing our config / setup rather than teach araund it. 🤔

Reply:
Nevermind!